### ISSUE TITLE
Revision of pip internals making the pip install stop after one package

### ISSUE THREAD

**Issue by kmbenitez** [nficano/python-lambda#126]
On my most recent invocations of lambda build I'm only getting the first package listed in my requirements.txt installed to the build. The output says that the rest are being processed, but the install is returning an error.

Given that Pip is actively trying to discourage use of itself as a package, I think it's time to begin using subprocess.

**Comment by winsome** [comment]
I'm seeing the same issue with python-lambda 3.2.4, pip 18.0, and the following requirements.txt

```
futures==3.2.0
jmespath==0.9.0
```

`futures` is installed into the zip whereas `jmespath` isn't.

**Comment by kmbenitez** [comment]
Same versions as @winsome. I just replicated the issue with the following requirements.txt

```
PyMySQL==0.8.1
arrow==0.4.2
```

and
```
arrow==0.4.2
PyMySQL==0.8.1
```
In both cases, the first package was installed into the zip, but the second wasn't.

I was able to update `_install_packages` (and the call to pip freeze) to use subprocess (introduced in Python 2.4). I'm not sure about the dependency implications though.

**Comment by kmbenitez** [comment]
I created a pull request a while back for another feature, and it appears my commit for the fix on this got pulled into that request. I'm not sure how to separate them to make two separate pull requests for the different features.

**Comment by henry034** [comment]
Same here. It will lose some packages due to the revision.

**Comment by soapergem** [comment]
Thanks to all reporting this issue. I have verified the bug using the arrow/PyMySQL requirements.txt file. I will test out the subprocess fix and if it works I'll try to get a release out sometime this week.

**Comment by kmbenitez** [comment]
Thanks! For my future knowledge, as far as I could tell I couldn't create a pull request because I already had one open. What's the right way to do this?

**Comment by soapergem** [comment]
@kmbenitez you probably did it correctly - for some reason I didn't see that you had a PR though. Forgive me, as I'm still relatively new to the Python ecosystem, so I may have just overlooked it. In any case, your changes have been merged in, and release v3.2.5 has been published which fixes this issue.
